### PR TITLE
Tip-of-the-Day, new tip: the full file path is shown in the status bar.

### PR DESCRIPTION
**Pull request by CWehli**

### CODE REVIEW

**Comment by fellen** [comment]
Can we insert it  more by context instead at the end?

File: doc/tip_of_the_day.list.c [hunk 1]
```diff
@@ -120,6 +120,11 @@ to a single account, start the search from that account's register.")
 in one of the tabs, select Window->New Window with Page \
 from the menu to duplicate that tab in a new window.")
 
+N_( "If you want to know at which directory the GnuCash file \
+is stored, hover over one of the entries in the history menu (File->\
```
Comment (fellen) [inline doc/tip_of_the_day.list.c:124]: your  recent GnuCash files are

**Comment by CWehli** [comment]
> Can we insert it more by context instead at the end?

Surely, where should the tip be placed? Maybe three tips further forward?

**Comment by fellen** [comment]
> Surely, where should the tip be placed? Maybe three tips further forward?

I think behind 
"You can easily import your existing financial data "
is a good place. Imagine a first time user: They have done something, but where is it gone?

Perhaps we should add a comment at the beginning of the file
`# Sort order: Imagine a first time user. When is your tip useful?`

File: doc/tip_of_the_day.list.c [hunk 2]
```diff
@@ -15,6 +17,11 @@ MS Money or other programs that export QIF files or OFX files. In the \
 File menu, click on the sub-menu Import and click on QIF or OFX file, \
 respectively. Then, follow the instructions provided.")
 
+ N_( "If you want to know at which directory your recent GnuCash files \
```
Comment (fellen) [inline doc/tip_of_the_day.list.c:20]: **in** which directory

@jralls proper US-English?

Comment (jralls) [reply doc/tip_of_the_day.list.c:20]: @fellen Yes, "in which directory".

File: doc/tip_of_the_day.list.c [hunk 3]
```diff
@@ -15,6 +17,11 @@ MS Money or other programs that export QIF files or OFX files. In the \
 File menu, click on the sub-menu Import and click on QIF or OFX file, \
 respectively. Then, follow the instructions provided.")
 
+ N_( "If you want to know in which directory your recent GnuCash files \
+are stored, hover over one of the entries in the history menu\n(File->\
+Most Recently Used-List).\nThe full path information is displayed \
```
Comment (fellen) [inline doc/tip_of_the_day.list.c:22]: Isn't `information`redundant?

**Review by fellen — COMMENTED** [review]
Sorry, each time i see something else.

**Comment by fellen** [comment]
I took the freedom to use
1. `directories` – each file can have another
2. square brackets around MRU – not all platforms have them in a submenu.

Thanks!

**Pull request merged**